aten/src/ATen/native/cpu/SumKernel.cpp

```diff
@@ -405,8 +405,7 @@ std::array<scalar_t, nrows> multi_row_sum(
     }
   }
 
-  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
-  std::array<scalar_t, nrows> ret;
+  std::array<scalar_t, nrows> ret{};
```

Comment:
This one is actually pretty safe to leave as a no_int since it's uninitialized. Better way to do this would be try to see if we could initialize it directly from a range from acc, but that seems like it would be unnecessarily complicated with the STL.

Reply:
It's possible to use multi-dimensional std::array for acc and then use std::array assignment here. But leave that change in a later PR.

Reply:
I would say leave it was before as initialing arrays are expensive and we are going to overwrite the value immediately.